config/locales/hyrax.en.yml

```diff
@@ -730,8 +734,8 @@ en:
         works:        "Your Works"
       nest_collections_form:
         create_within:          "'%{child_title}' has been added to '%{parent_title}'"
-        create_under:          "'%{child_title}' has been added to '%{parent_title}'"
-        create_under_failed:   "'%{child_title}' cannot be added to '%{parent_title}'"
```

Comment:
Did you intend to remove create_under_failed?

Reply:
I found nothing using this translation so i removed it.